src/Notes/WC_Admin_Notes_Review_Shipping_Settings.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * WooCommerce Admin Review Shipping Settings Note Provider.
+ *
+ * Adds an admin note prompting the admin to review their shipping settings.
+ *
+ * @package WooCommerce Admin
+ */
+
+namespace Automattic\WooCommerce\Admin\Notes;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC_Admin_Notes_Review_Shipping_Settings
+ */
+class WC_Admin_Notes_Review_Shipping_Settings {
+	use NoteTraits;
+
+	const NOTE_NAME = 'wc-admin-review-shipping-settings';
+
+	/**
+	 * Possibly add note.
+	 */
+	public static function possibly_add_note() {
```

Comment:
This looks to be the same as the note traits method.  Can we remove this method or does it serve a different purpose?

Reply:
Not sure how that happened :-$ fixed